rocks.thorium.Mercury.metainfo.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>rocks.thorium.Mercury</id>
+  <launchable type="desktop-id">rocks.thorium.Mercury.desktop</launchable>
+  <name>Mercury</name>
+  <developer_name>Alex F.</developer_name>
+  <summary>Compiler optimized, private Firefox fork</summary>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>MPL-2.0</project_license>
+  <description>
+    <p>Firefox fork with compiler optimizations and patches from Librewolf, Waterfox, Ghostery, and BetterFox.</p>
+    <p>It aims to be the Firefox equivalent of my main project: Thorium (a Chromium fork).</p>
+  </description>
+  <releases>
```

Comment:
Screenshots missing https://docs.flathub.org/docs/for-app-authors/metainfo-guidelines/#screenshots